processing/src/test/java/org/apache/druid/query/groupby/GroupByQueryRunnerTest.java

```diff
@@ -13682,4 +13755,16 @@ private void cannotVectorize()
       expectedException.expectMessage("Cannot vectorize!");
     }
   }
+
+  private void verifyMetrics(long queries, boolean skipMergeDictionary)
```

Comment:
What is skipMergeDictionary and when is it required?

Reply:
Renamed, this is basically to skip verifying the merge dictionary size metric, since some of the queries aggregate on non-string dimensions.